src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/TargetCompatibilityReferenceAnalyzer.cs

```diff
@@ -31,23 +31,23 @@ public async Task<PackageAnalysisState> AnalyzeAsync(IProject project, PackageAn
                 throw new ArgumentNullException(nameof(state));
             }
 
-            var targetFramework = _tfmSelector.SelectTFM(project.Required());
```

Comment:
I wonder if we would want to have it find the minimum version that supports both the selected tfm and the current (if possible)

Reply:
Not blocker for now, but may be nice

Reply:
def nice to have :)